test/schema/pull.mjs

```diff
@@ -194,7 +194,8 @@ describe("schema pull", function () {
     expect(logger.stdout).to.have.been.calledWith("Change cancelled");
     expect(fs.writeFile).to.have.not.been.called;
     expect(fsp.unlink).to.have.not.been.called;
-    expect(fs.mkdirSync).to.have.not.been.called;
+    // Called twice during Credentials initialization, but not called during the pull command
+    expect(fs.mkdirSync).to.have.been.calledTwice;
```

Comment:
Would like to "reset" the stub or somehow separate the `fs` from how it's used in middleware vs in the actual command helper. I imagine it'll be a pain this way to have to keep track of every way that middleware has already interacted with stubs...

Reply:
yeah - this is an early test and i think the better pattern is to assert that the call you care about happened (`expect(...).to.have.been.calledWith(...)`). this case is a little funky because we want to assert the opposite (not called with...).